docker-compose.yml

```diff
@@ -50,7 +50,7 @@ services:
       - "8080:80"
     environment:
       - MHS_LOG_LEVEL=NOTSET
-      - MHS_SDS_URL=ldap://msg.opentest.hscic.gov.uk
+      - MHS_SDS_URL=ldap://192.168.128.11
```

Comment:
I have to make this change for it to work. Does anyone else have DNS working on their local VPN?

Reply:
maybe adding one into etc/hots will help

Reply:
like:
`192.168.128.11 msg.opentest.hscic.gov.uk`

Reply:
Might was well just use the IP address here then? OpenTest certs can't be validated anyway.